### PR TITLE
PC-2274 - Braze javascript interface for android custom webviews

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

**Description of Change:**
https://jira.braze.com/browse/PC-2274

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+nav_title: Braze Javascript Interface
+platform: Android
+page_order: 9
+
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:5]: extra space?

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+## Adding The Braze Javascript Interface to Webviews
+
+Using Braze functionality within a custom WebView in your app can be done by adding the Braze javascript interface. Then, the same API used with [HTML In-App Messages][1] can be used within your custom WebView.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:9]: Using Braze functionality from a WebView in your app can be done by adding the Braze JavaScript interface to your WebView as shown below. Once the interface has been added, the same API available for [HTML In-App Messages][1] will be available within your custom WebView.

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+nav_title: Braze Javascript Interface
+platform: Android
+page_order: 9
+
+---
+## Adding The Braze Javascript Interface to Webviews
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:7]: JavaScript

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md (hunk 1 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:5]: All the jekyll headers are like that.

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md (hunk 2 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:9]: changed

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md (hunk 3 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:7]: changed

**Comment by radixdev** [comment]
@KellieHawks ready for merge

File: _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/android/advanced_use_cases/adding_braze_js_interface.md:5]: At least some aren't, e.g. https://github.com/Appboy/braze-docs/blame/develop/_docs/_developer_guide/platform_integration_guides/xamarin/ios/inapp_messages.md#L5 but not sure it matters

**Pull request merged**